src/pages/Profile.tsx

```diff
@@ -21,45 +23,176 @@ import { SubscriptionCollection } from "../../generated/definitions/backend/Subs
+              <div>
+                <span className="circle circle-red" />
+                <div>
+                  <span className="light-text">
+                    {t("service:state")}:&nbsp;
+                  </span>
+                  <span className="dark-text">
+                    {t("profile:service_not_valid")}
+                  </span>
+                </div>
+              </div>
+            </div>
+          ) : (
+            ""
+          )}
+          {status === ServiceStatus.REVIEW ? (
+            <div className="service-status">
+              <div>
+                <span className="circle circle-yellow" />
+                <div>
+                  <span className="light-text">
+                    {t("service:state")}:&nbsp;
+                  </span>
+                  <span className="dark-text">
+                    {t("profile:service_review")}
+                  </span>
+                </div>
+              </div>
+            </div>
+          ) : (
+            ""
+          )}
+          {status === ServiceStatus.VALID ? (
+            <div className="service-status">
+              <div>
+                <span className="circle circle-green" />
+                <div>
+                  <span className="ligh-text">{t("service:state")}</span>:
+                  &nbsp;
+                  <span className="dark-text">
+                    {t("profile:service_valid")}
+                  </span>
+                </div>
+              </div>
+            </div>
+          ) : (
+            ""
+          )}
+          {status === ServiceStatus.DEACTIVE ? (
+            <div className="service-status">
+              <div>
+                <span className="circle circle-green" />
+                <div>
+                  <span className="ligh-text">{t("service:state")}</span>:
+                  &nbsp;
+                  <span className="dark-text">
+                    {t("profile:service_valid")} (
+                    {t("service:deactive_service_title")})
+                  </span>
+                </div>
+              </div>
+            </div>
+          ) : (
+            ""
+          )}
+          {status === ServiceStatus.LOADING ? (
+            <div className="service-status">
+              <div>
+                <span className="circle circle-black" />
+                <div>
+                  <span className="ligh-text">{t("service:state")}</span>:
+                  &nbsp;
+                  <span className="dark-text">
+                    {t("profile:service_loading")}
+                  </span>
+                </div>
+              </div>
+            </div>
+          ) : (
+            ""
+          )}
+        </div>
+        <div className="col-md-4">
+          <Link
+            className="btn btn-primary"
+            to={{
+              pathname: `/service/${service.service_id}`,
+              state: {
+                isVisible: atLeastOneVisible
```

Comment:
Should this be
```suggestion
                isVisible: !!service.is_visible
```

Reply:
I will remove this prop because the Jira backend don't use it anymore.

Reply:
Fix with 3819851